internal/testing/s3_test.go

```diff
@@ -129,6 +129,9 @@ func TestS3Basic(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	if res.LastModified.IsZero() {
```

Comment:
Could you update the timestamp checks to compare against the time of when the test started? `IsZero` might not catch all cases since we used to set it to Unix in some spots.